examples/applications/slack/configuration.yaml

```diff
@@ -0,0 +1,25 @@
+#
+#
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+configuration:
+  resources:
```

Comment:
you don't need this, it is enough to use the secrets

if you want you can use a "global variable" and you can define it here, see the feature here (it is not documented yet probably)
https://github.com/LangStream/langstream/pull/499

```
defaults:
         globals:
            slack:
                 url: "https://xxxx"
```

Then people will be able to override it with a custom "instance.yaml" file